open3d_slam_ros/launch/mapping.launch

```diff
@@ -10,6 +10,7 @@
 	<arg name="map_saving_folder" default="$(find open3d_slam_ros)/data/maps/"/>
 	<arg name="num_accumulated_range_data" default="1"/>
 	<arg name="is_read_from_rosbag" default="false"/>
+	<arg name="is_initialize_map" default="false"/>
```

Comment:
I've added it here and not to the param file as asked in the previous comment since it is a ros flag. For me there is a clear distinction with parameters between Slam and Slam_ros and the is_initialize_map is a ros functionality.